### PR TITLE
youtube-viewer: add page

### PR DESCRIPTION
**Pull request by g-w1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zdroid** [comment]
It seems this can run on OSX too. I suggest moving this to `common/`.

File: pages/common/youtube-viewer.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# youtube-viewer
+
+> A command line tool to let you watch youtube.
+> More information: <https://github.com/trizen/youtube-viewer>.
+
+- Search for a video:
+
+`youtube-viewer {{search term}}`
```
Comment (zdroid) [inline pages/common/youtube-viewer.md:8]:
```suggestion
`youtube-viewer {{search_term}}`
```

File: pages/common/youtube-viewer.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Login to your youtube account:
```
Comment (zdroid) [inline pages/common/youtube-viewer.md:10]:
```suggestion
- Login to your YouTube account:
```

File: pages/common/youtube-viewer.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`youtube-viewer --login`
+
+- Watch a video with a specific url in vlc:
```
Comment (zdroid) [inline pages/common/youtube-viewer.md:14]:
```suggestion
- Watch a video with a specific URL in VLC:
```

File: pages/common/youtube-viewer.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`youtube-viewer --player={{vlc}} {{https://youtube.com/watch?v=1dastnond}}`
+
+- Get to a search prompt and play the video that is selected in 720p:
```
Comment (zdroid) [inline pages/common/youtube-viewer.md:18]:
```suggestion
- Get to a search prompt and play the selected video in 720p:
```
What do you mean by `get to a search prompt`?

File: pages/common/youtube-viewer.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# youtube-viewer
+
+> A command line tool to let you watch youtube.
```
Comment (zdroid) [inline pages/common/youtube-viewer.md:3]:
```suggestion
> Command-line application for searching and playing videos from YouTube.
```
This is more like the tool's own description, which is usually preferrable.

File: pages/common/youtube-viewer.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Login to your YouTube account:
+
+`youtube-viewer --login`
+
+- Watch a video with a specific URL in VLC:
+
+`youtube-viewer --player={{vlc}} {{https://youtube.com/watch?v=1dastnond}}`
+
+- Get to a search prompt and play the selected video in 720p:
```
Comment (sbrl) [inline pages/common/youtube-viewer.md:18]: I'm confused by this. What does `Get to a search prompt` mean specifically?

Comment (g-w1) [reply pages/common/youtube-viewer.md:18]: Something like this:
image

File: pages/common/youtube-viewer.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`youtube-viewer -7`
```
Comment (agnivade) [inline pages/common/youtube-viewer.md:20]: 7 should be in tokens(`{{}}`)

Comment (zdroid) [reply pages/common/youtube-viewer.md:20]: If it's related to 720p, then yes.

Comment (g-w1) [reply pages/common/youtube-viewer.md]: Ok added ``-{{7}}``

Comment (g-w1) [reply pages/common/youtube-viewer.md]: Yes it is related to 720p.

File: pages/common/youtube-viewer.md (hunk 6 above)
Comment (andrik) [reply pages/common/youtube-viewer.md:18]: @sbrl Does that clarify?

Comment (einverne) [inline pages/common/youtube-viewer.md:18]:
```suggestion
- Get a search prompt and play the selected video in 720p:
```
Does this sound more nature?

Comment (zdroid) [reply pages/common/youtube-viewer.md:18]: or `Open a search prompt`?

Comment (einverne) [reply pages/common/youtube-viewer.md:18]: `get` or `open` is okay with me.

Comment (sbrl) [reply pages/common/youtube-viewer.md:18]: I agree with @zdroid - `Open` sounds better.
```suggestion
- Open a search prompt and play the selected video in 720p:
```

Comment (g-w1) [reply pages/common/youtube-viewer.md:18]: It is a cli search.

Comment (g-w1) [reply pages/common/youtube-viewer.md]: Changed to display.

**Comment by zdroid** [comment]
Since all concerns were resolved, I'm merging this.

File: pages/common/youtube-viewer.md (hunk 6 above)
Comment (sbrl) [reply pages/common/youtube-viewer.md]: Ah, I see! Sorry, this PR slipped under the radar.  It seems like this comment has been resolved further down.

**Pull request merged**